src/shims/unix/foreign_items.rs

```diff
@@ -58,6 +58,16 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             // 512 seems to be a reasonable default. The value is not critical, in
             // the sense that getpwuid_r takes and checks the buffer length.
             ("_SC_GETPW_R_SIZE_MAX", |this| Scalar::from_int(512, this.pointer_size())),
+            // the machine's fd table does not start with a pre-allocated size but just the
+            // standard streams so we can't use it here and the spec imposes
+            // a minimum of `_POSIX_OPEN_MAX` (20).
+            ("_SC_OPEN_MAX", |this| Scalar::from_int(20, this.pointer_size())),
+            // The spec imposes a minimum of `_POSIX_ARG_MAX` (4096).
+            ("_SC_ARG_MAX", |this| Scalar::from_int(4096, this.pointer_size())),
```

Comment:
We don't support exec anyway so I don't think there's any point in supporting this constant.